meta-dts-distro/recipes-bsp/txe-secure-boot/txesbmantool_git.bb

```diff
@@ -12,7 +12,7 @@ SRC_URI = " \
     git://github.com/Dasharo/coreboot.git;branch=txe_sb_tool;protocol=https \
 "
 
-SRCREV = "235c946838dc8c619ff821c9791386d63f5cbd6a"
+SRCREV = "98e264b20ac2cabd1c8eaaae99113dcdc3cf52c7"
```

Comment:
This change has nothing to do with this PR

Reply:
It still needs to be done, so I figured I might as well do it now.

Reply:
I dropped the commit